core/metacling/src/TCling.cxx

```diff
@@ -5897,6 +5898,12 @@ static void* LazyFunctionCreatorAutoloadForModule(const std::string& mangled_nam
          if (!is_directory(DirPath))
             continue;
 
+         if (DirPath.endswith("/"))
```

Comment:
Could you use the llvm's path separator here?

Reply:
http://llvm.org/doxygen/namespacellvm_1_1sys_1_1path.html#aa767afa14fa4c9059e62822bf965723a
I'm doing this because of this, path ends with "/" when `/foo       => /`

Reply:
How that would work on win where the separator is different?